app/serializers.py

```diff
@@ -6,6 +6,7 @@
 class EmployeeListSerializer(serializers.ModelSerializer):
     first_name = serializers.CharField(source="user.first_name", read_only=True)
     last_name = serializers.CharField(source="user.last_name", read_only=True)
+    image = serializers.ImageField(source="user.image", read_only=True)
```

Comment:
The URL returned doesn't work for the default image.
Maybe update the default path in the model.

Reply:
![image](https://user-images.githubusercontent.com/90555439/202118577-201fb802-0e2f-4db7-ac19-af16e28521fc.png)
It's working on both cases, first one is default image second one is custom

Reply:
It is working for your directory structure.
You have added static on your end.